unified-scheduler-pool/src/lib.rs

```diff
@@ -1575,6 +1575,13 @@ mod tests {
 
         // See the old (= idle) scheduler gone only after solScCleaner did its job...
         sleepless_testing::at(&TestCheckPoint::AfterIdleSchedulerCleaned);
+
+        // The following assertion is racy.
+        //
+        // We need to make sure new_scheduler isn't treated as idle up to now since being returned
+        // to the pool after sleep(SHORTENED_TIMEOUT_DURATION * 10).
```

Comment:
is `SHORTENED_TIMEOUT_DURATION` the same as `SHORTENED_MAX_POOLING_DURATION`?

Reply:
oops, nice catch of bad copy paste... 0997fab5a46ac683f215383f0de8e7a2afba7426